gui/preload-main.js

```diff
@@ -3164,6 +3184,7 @@ function set_setting_switch(setting_switch, enable_object, state) {
   enable_setting(setting_switch, enable_object);
 }
 
+var rigctlActive = false;
```

Comment:
Suggestion: naming this "rigctld" so we're avoiding confusion with "rigctl" which is also able doing network stuff ( as client )

Reply:
Ok, will rename.